CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ Then add comments on pull request 😉.
 
 ## How-to work with GitHub and Pull Requests
 
-You can develop directly on your rabbit (RasperryPi) that you have of course
+You can develop directly on your rabbit (Rasperry Pi) that you have of course
```

Comment:
Rasp**b**erry the b is missing!

Reply:
It was already missing...
I assumed this was some secret code shared between the previous editors of the document, and I didn't dare change it.